amplipi/models.py

```diff
@@ -71,6 +71,8 @@ class fields(SimpleNamespace):
   Volume = Field(ge=MIN_VOL_DB, le=MAX_VOL_DB, description='Output volume in dB')
   VolumeF = Field(ge=MIN_VOL_F, le=MAX_VOL_F,
                   description='Output volume as a floating-point scalar from 0.0 to 1.0 representing MIN_VOL_DB to MAX_VOL_DB')
+  VolumeDeltaF = Field(ge=(MAX_VOL_F * -1), le=MAX_VOL_F,
+                       description='Output volume as a floating-point scalar from -1.0 to 1.0 representing the distance between the current and goal volume')
```

Comment:
Nitpick but Maybe "adjustment to output volume"? This delta is not a volume by itself.